resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/Client.scala

```diff
@@ -120,6 +120,7 @@ private[spark] class Client(
         .withName(kubernetesDriverPodName)
         .addToLabels(allLabels.asJava)
         .addToAnnotations(parsedCustomAnnotations.asJava)
+        .addToAnnotations(SPARK_APP_NAME_ANNOTATION, appName)
```

Comment:
Doesn't seem that way since we put in full init-container specs in the annotations.